CHANGELOG.md

```diff
@@ -43,6 +43,23 @@ system.io {
 }
 ```
 
+* Added a `script-epilogue` configuration option to adjust the logic that runs at the end of the scripts which wrap call executions.
+  This option is adjustable on a per-backend basis.  If unspecified, the default value is `sync`.  To turn off the default `sync` 
+  behavior on the `Local` backend:
```

Comment:
Or just a bigger comment in `reference.conf` as a self-documenting file? And then reference that from here?